components/webext-storage/src/store.rs

```diff
@@ -29,7 +29,7 @@ use serde_json::Value as JsonValue;
 /// connection with our sync engines - ie, these engines also hold an Arc<>
 /// around the same object.
 pub struct WebExtStorageStore {
-    db: Arc<ThreadSafeStorageDb>,
+    pub db: Arc<ThreadSafeStorageDb>,
```

Comment:
Can we use `pub(crate)` here assuming it works? It probably doesn't matter at all in practice, but it's closer to the intent.